core/client/fs/src/main/java/alluxio/client/file/AlluxioFileOutStream.java

```diff
@@ -142,6 +145,8 @@ public void close() throws IOException {
       }
 
       CompleteFilePOptions.Builder optionsBuilder = CompleteFilePOptions.newBuilder();
+      optionsBuilder.setCommonOptions(FileSystemMasterCommonPOptions.newBuilder()
+          .setOperationId(new OperationId(UUID.randomUUID()).toFsProto()).buildPartial());
```

Comment:
how fast is `UUID.randomUUID()`? Should we worry about this being called very frequently since it's in a somewhat hot code path? does it use a normal `Random` instance or `ThreadLocalRandom` under the hood?

Reply:
I remember seeing complaints that UUID is slow in Java, [this](http://braveo.blogspot.com/2013/05/uuidrandomuuid-is-slow.html) is not alone. 
I think essentially we are using UUID to guard the operations affected in a master flip, so global uniqueness is not a must. We just want uniqueness in the active operations right? That can be an argument favoring `ThreadLocalRandom`.

Reply:
If we just need uniqueness across active operations, we may also reduce the size of OpId from 2 int64 to just one. Well the performance difference will not be too much though.

Reply:
We want uniqueness across all clients of Alluxio. I don't see how can we achieve this other than using `UUID`.

Reply:
As for speed, I haven't seen any degradation on overall master throughput.